account-kit/core/src/actions/setChain.ts

```diff
@@ -26,7 +26,15 @@ export async function setChain(config: AlchemyAccountsConfig, chain: Chain) {
     throw new ChainNotFoundError(chain);
   }
 
-  await switchChain(config._internal.wagmiConfig, { chainId: chain.id });
+  const chainId = getChainId(config._internal.wagmiConfig);
```

Comment:
nit: i might prefer to name this something like `wagmiChainId` to avoid confusion

Reply:
not blocking on this though, just a suggestion